Source/trigs.cpp

```diff
@@ -903,7 +903,7 @@ void CheckTriggers()
 					}
 
 					InitDiabloMsg(abortflag);
-					NetSendCmdLoc(true, CMD_WALKXY, x, y);
+					NetSendCmdLoc(myplr, true, CMD_WALKXY, x, y);

```

Comment:
@galaxyhaxz do you know how psx dealt with this?

Reply:
Since they didn't need networking, they got rid of sending packets and instead it just directly tries a packet always using `myplr` as a hardcoded id. In the `ProcessPlayers` function, it loops through twice setting `myplr` to both 0 then 1.